app/components/preview/BlogPostPreview.tsx

```
@@ -0,0 +1,317 @@
+                  <Badge key={index} variant="secondary" className="bg-yellow-100 text-yellow-800">
+                    {keyword}
+                  </Badge>
+                ))}
+              </div>
+            </div>
+          )}
+          
+          {/* Blog Content with Keyword Highlighting */}
+          <div 
+            className="prose prose-lg max-w-none text-gray-800 leading-relaxed"
+            dangerouslySetInnerHTML={{ 
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)